### DISCUSSION TITLE
Is the total count of a type of entity exposed within the GraphQL interface?

### DISCUSSION THREAD

**Discussion by nathaniellee** [5e-bits/5e-srd-api#581] [Q&A] [answered]
Let's say I want to create a table view of spells with the ability to navigate to previous or next pages based on the number of spells I'm showing per page and the total count of spells in the system. Is there a way to pull said total count via GraphQL? I see that it's part of the response when just calling the API in standard fashion, but I don't see the same in the GraphQL response. If I want to be able to get this information, do I just need to make one call via the standard endpoint to get `count` and then my actual call via GraphQL to get the data I'm looking for?

**Comment by bagelbits** [comment, marked as answer]
Unfortunately, there is not. You would either have to make a call to the API, or gather all of the spells and then count them.

Or you could set your limit to a really big number and then just count and paginate on the frontend.

**Reply by nathaniellee** [reply]
Thanks for confirming! I went with the route of making a standard call to the REST APIs to grab count there. No worries. Just wondering! :)
